### PR TITLE
feat: collapsible sidebar and asset relocation exp

### PR DESCRIPTION
**Pull request by sarojgurung**

### CODE REVIEW

File: docs/components/buttons-and-links/button.md [hunk 1]
```diff
@@ -24,7 +24,14 @@ There are several types of Buttons with varying degree of emphasis. Emphasis det
 
 ## Sizes
 
-![Button size](/img/buttons/button-size.png)
+![Button size](../../assets/button-size.png)
```
Comment (danielwang) [inline docs/components/buttons-and-links/button.md]: I am not sure when building the site, webpack will load the images. I will try it out.

File: docusaurus.config.js [hunk 2]
```diff
@@ -50,6 +50,12 @@ module.exports = {
         // },
       ],
     },
+    docs: { 
+      sidebar: {
+        autoCollapseCategories: true,
```
Comment (danielwang) [inline docusaurus.config.js:55]: we can have it enabled for now, we may want to turn it off in the future.

Comment (sarojgurung) [reply docusaurus.config.js:55]: yup sure. its just experimentation for the time being

File: docs/components/buttons-and-links/button.md (hunk 1 above)
Comment (sarojgurung) [reply docs/components/buttons-and-links/button.md]: understood

Comment (danielwang) [reply docs/components/buttons-and-links/button.md]: I pulled your feature branch and tested, I think it works fine. you can move the related images from img folder to here

**Comment by danielwang** [comment]
LGTM, please land it

btw, please note this is refactor as opposed to chore. **No action required.**

**Pull request merged**